Sources/SnowManLogic/ViewLogic/IO.swift

```diff
@@ -0,0 +1,18 @@
+public class IO: Receivable {
+
+	public init() {
+	}
+
+	public func getUserInput(getInput: () -> String) -> String {
+		return String(getInput())
+	}
+
+	public func myReadLine() -> String {
+		return readLine(stripNewline: true) ?? ""
+	}
```

Comment:
This branching logic is untestable, you should fix it so that it only wraps the `readLine` function and that's it.